.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: Release
+on:
+  push:
+    branches: [master]
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: olafurpg/setup-scala@v2
+      - uses: olafurpg/setup-gpg@v2
+      - run: git fetch --tags --unshallow
+      - name: Publish ${{ github.ref }}
+        run: sbt ci-release docs/docusaurusPublishGhpages
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```

Comment:
@olafurpg Hello! I have not access to repo settings. Can you please check are this secrets specified?

Reply:
These four variables should already be configured for this repo but I think you are still missing some secret variables for the deploy key to publish the website.

Reply:
I added missing variables just like in scalameta repo

Reply:
is there a way to test it, other than merging and trying to push a release?

Reply:
Release job pushes snapshot to the [sonatype repo](https://oss.sonatype.org/content/repositories/snapshots/org/scalameta/scalafmt-core_2.13/) on every merge. So we can test sonatype publishing process after merge.

But I don't know how to test website publishing

Reply:
if there's no better way, let's try merging. if necessary, we create some test release, out of order with the proper sequence.

Reply:
There’s no way to test this via a PR but you can merge into a non-master branch. I normally just merge and push fixes into master until it works. 

The google credentials are not needed for the website, only the deploy key.